otter/worker/launch_server_v1.py

```diff
@@ -325,7 +340,11 @@ def _create_server():
 
         If not, and if no further errors occur, server creation can be retried.
         """
-        d = create_server_sem.run(_create_with_delay)
+        sem = get_sempahore("create_server", "worker.create_server_limit")
```

Comment:
I think a better a option is avoid strings altogether and use some kind of enums. However, this is old code supposed to go away. So I am feeling lazy to make that change :).